src/k8s/pkg/client/dqlite/remove.go

```diff
@@ -20,6 +20,10 @@ func (c *Client) RemoveNodeByAddress(ctx context.Context, address string) error
 		return fmt.Errorf("failed to retrieve cluster nodes")
 	}
 
+	if len(members) == 1 {
```

Comment:
exit early if there is only one node in the cluster.
Otherwise, this would unnecessarily retry and block further down this function